tests/unit/services/test_tunnel_service.py

```diff
@@ -24,6 +24,6 @@ def test_create_tunnel_user(self, current_free_user, session):
                 TunnelCreationService(
                     current_user=current_free_user,
                     subdomain_id=asub.id,
-                    port_type=["http"],
+                    port_types=["http"],
```

Comment:
Did we change the JSON Schema for tunnel create to match this?

Reply:
It is still port in schema. It would be a breaking change so going to hold off doing it for right now